CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/experimental/TestParametersPluginNGTest.java

```diff
@@ -0,0 +1,494 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.dataaccess.plugins.experimental;
+
+import au.gov.asd.tac.constellation.graph.GraphElementType;
+import au.gov.asd.tac.constellation.graph.node.plugins.DefaultPluginInteraction;
+import au.gov.asd.tac.constellation.graph.processing.GraphRecordStore;
+import au.gov.asd.tac.constellation.graph.processing.GraphRecordStoreUtilities;
+import au.gov.asd.tac.constellation.graph.processing.RecordStore;
+import au.gov.asd.tac.constellation.graph.schema.analytic.concept.AnalyticConcept;
+import au.gov.asd.tac.constellation.graph.schema.analytic.concept.TemporalConcept;
+import au.gov.asd.tac.constellation.plugins.PluginException;
+import au.gov.asd.tac.constellation.plugins.PluginNotificationLevel;
+import au.gov.asd.tac.constellation.plugins.parameters.PluginParameter;
+import au.gov.asd.tac.constellation.plugins.parameters.PluginParameters;
+import au.gov.asd.tac.constellation.plugins.parameters.types.DateTimeRange;
+import au.gov.asd.tac.constellation.plugins.parameters.types.SingleChoiceParameterType;
+import au.gov.asd.tac.constellation.views.dataaccess.CoreGlobalParameters;
+import au.gov.asd.tac.constellation.views.dataaccess.DataAccessPluginCoreType;
+import au.gov.asd.tac.constellation.views.dataaccess.plugins.experimental.TestParametersPlugin.GraphElementTypeParameterValue;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * @author aldebaran30701
+ */
+public class TestParametersPluginNGTest {
+
+    public TestParametersPluginNGTest() {
+        // Intentionally left blank
+    }
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+        // Intentionally left blank
+    }
+
+    @AfterClass
+    public static void tearDownClass() throws Exception {
+        // Intentionally left blank
+    }
+
+    @BeforeMethod
+    public void setUpMethod() throws Exception {
+        // Intentionally left blank
+    }
+
+    @AfterMethod
+    public void tearDownMethod() throws Exception {
+        // Intentionally left blank
+    }
+
+    /**
+     * Test of getType method, of class TestParametersPlugin.
+     */
+    @Test
+    public void testGetType() {
```

Comment:
Just a comment but I reckon the likes of getType, getPosition, and getDescription probably don't need a test as they're one line functions that return the same value everytime (so no real value added from having these unit tests)

Reply:
Would you like me to remove them? They were autogenerated and I decided to spend the few seconds implementing something while it was there.

Reply:
They're fine to be left there. They're not doing any damage really